Gemfile

```diff
@@ -9,6 +9,7 @@ gem "bootsnap", ">= 1.4.2", require: false
 gem "bootstrap-sass", "3.4.1"
```

Comment:
ý tưởng code chức năng này là gì?

Reply:
Người dùng đã có thể đăng ký từ Chap7. nên chap 8 sẽ hoàn thành chức năng đăng nhập và đăng xuất cơ bản. Ứng dụng sẽ duy trì trạng thái đăng nhập cho đến khi trình duyệt được đóng hoặc đăng xuất. Mỗi phiên như vậy gọi là session. Tạo sign in form với form for, tạo 1 session khi đăng nhập, dẫn tới trang hiển thị user và xóa session, dẫn tới trang chủ khi đăng xuất.

Reply:
đây la ý tưởng code của chap 8